### ISSUE TITLE
General Design and Layout Ideas

### ISSUE THREAD

**Issue by SnoutBug** [SnoutBug/mmcklauncher#3]
Since the Launcher has the user's avatar sticking out of bounds I've not yet included a top down version. I'm looking for a proper solution

**Comment by Vistaus** [comment]
It would also be great if the launcher could be centered on screen. And by that I mean true center, not the bottom center that's already available. That would make me want to use as a full replacement for the current one I'm using (Ditto). 🙂

**Comment by SnoutBug** [comment]
Moved from #9 by [davigamer987](https://github.com/davigamer987)
> So i have been looking for a kde launcher for years that floats above the panel and that looks good (I don like the windows 11 clones), I Attached a poorly made "concept" on how that would look, idk how to make rounded corners tho
![koolde](https://user-images.githubusercontent.com/68574209/156694495-eaeae9e4-3e6d-4761-bf54-c0ac4f9b866f.png)

**Comment by SnoutBug** [comment]
Added to release [0.3](https://github.com/SnoutBug/mmcklauncher/releases/tag/0.3)

**Comment by Vistaus** [comment]
> Added to release [0.3](https://github.com/SnoutBug/mmcklauncher/releases/tag/0.3)

Thanks. I just tried it, but it doesn't open in the center of the screen. The setting for making it so keeps reverting to “Default”. :/

**Comment by davigamer987** [comment]
I know this is nitpicking, but is it possible to add a slider or option to make the launcher float a bit higher up in settings?

**Comment by SnoutBug** [comment]
> > Added to release [0.3](https://github.com/SnoutBug/mmcklauncher/releases/tag/0.3)
> 
> Thanks. I just tried it, but it doesn't open in the center of the screen. The setting for making it so keeps reverting to “Default”. :/

You might have to restart your pc for the changes to properly load. I had to with my laptop

**Comment by Vistaus** [comment]
@SnoutBug You're right. I did that and now it does work. Thanks for the suggestion. 🙂

**Comment by SnoutBug** [comment]
> I know this is nitpicking, but is it possible to add a slider or option to make the launcher float a bit higher up in settings?

 @davigamer987 I've added a the option for a custom offset when the panel is floating in the latest release ([0.4](https://github.com/SnoutBug/mmcklauncher/releases/tag/0.4))

**Comment by davigamer987** [comment]
> > I know this is nitpicking, but is it possible to add a slider or option to make the launcher float a bit higher up in settings?
> 
> @davigamer987 I've added a the option for a custom offset when the panel is floating in the latest release ([0.4](https://github.com/SnoutBug/mmcklauncher/releases/tag/0.4))

![image](https://user-images.githubusercontent.com/68574209/157320845-b6ea5138-58e6-469d-9e77-bc7dbabe300b.png)
The user image no longer shows in new version, i restarted my pc and it did not fix it

**Comment by davigamer987** [comment]
@SnoutBug

**Comment by SnoutBug** [comment]
moved to seperate issue

**Comment by SuperDog34** [comment]
Hey, I really appreciate your work, in my opinion it's the most beatiful App Launcher for KDE. The only problem, in my opinion, is that there are more practical. My idea and solution would be to put the categories in tabs instead of this click system.
Thanks

**Comment by nBalykin** [comment]
> Hey, I really appreciate your work, in my opinion it's the most beatiful App Launcher for KDE. The only problem, in my opinion, is that there are more practical. My idea and solution would be to put the categories in tabs instead of this click system. Thanks

Totally agree with you! 



And also for me there is little bug. I've removed some categories from KMenuEdit and they messed up (check screenshots).
![1](https://user-images.githubusercontent.com/42620164/167222002-9ebe479e-5052-465f-b197-b978816c080e.png)
![2](https://user-images.githubusercontent.com/42620164/167222005-26a407e2-2ad4-49dd-b6b3-2ee859b68f95.png)
![3](https://user-images.githubusercontent.com/42620164/167222008-31ee1e85-4878-4712-91ce-6da13583749a.png)
 


And one more thing. 
I really want to see an extended reboot menu. My machine 90% of the time does not turn off, but goes to sleep. And now, for the sake of this, I had to attach a shutdown widget to the taskbar. And I want it to work right in the application menu.

**Comment by SnoutBug** [comment]
> And also for me there is little bug. I've removed some categories from KMenuEdit and they messed up (check screenshots).

I've created issue #27 

> really want to see an extended reboot menu. My machine 90% of the time does not turn off, but goes to sleep. And now, for the sake of this, I had to attach a shutdown widget to the taskbar. And I want it to work right in the application menu.

Pressing the shutdown button should open a selection of options with "shutdown" being selected as default. There should be a 60 second time window to change that selection

**Comment by nBalykin** [comment]
> Pressing the shutdown button should open a selection of options with "shutdown" being selected as default. There should be a 60 second time window to change that selection ![proxy-image](https://user-images.githubusercontent.com/71790678/167243086-2bafb5a8-787f-4a1e-922e-b5e9ce19e32b.png)

I do not have this menu. Also I can't found any settings of this in kde settings.

**Comment by StarrKiss** [comment]
The ability to change fonts would be greatly appreciated.

**Comment by StarrKiss** [comment]
Also, it's worth noting that the matching theme has a bug when the buttons/selection are light, like so:
![image](https://user-images.githubusercontent.com/13007697/171064506-c553ff03-2844-4081-9529-a792c920d9dd.png)
Compare to in system settings:
![image](https://user-images.githubusercontent.com/13007697/171064526-1ab41977-dc23-45ce-afdd-20233c113a6c.png)
